forms.rst

```diff
@@ -542,11 +542,11 @@ the guessed field.
 .. index::
    single: Forms; Field type guessing
 
-Field Type Options Guessing
+Field Attributes Guessing
```

Comment:
I understand why you did it but this change is not needed since guessers are about to change type's options, actually `required` and `maxlength` are define thanks to the `attr` option in the form factory but the registered doctrine and validation guessers are setting some options too, maybe it's worth documenting them though. I don't know what others think?